include-build/roborazzi-gradle-plugin/src/main/java/io/github/takahirom/roborazzi/GenerateRobolectricPreviewTestsExtension.kt

```diff
@@ -24,6 +25,14 @@ open class GenerateRobolectricPreviewTestsExtension @Inject constructor(objects:
   val enable: Property<Boolean> = objects.property(Boolean::class.java)
     .convention(false)
   val packages: ListProperty<String> = objects.listProperty(String::class.java)
+  val customTestClassFQDN: Property<String> = objects.property(String::class.java)
+    .convention("com.github.takahirom.roborazzi.DefaultRobolectricPreviewTest")
+  val robolectricConfig = objects.mapProperty(String::class.java, String::class.java)
+    .convention(
+      mapOf(
+        "sdk" to "[33]"
```

Comment:
Thanks. I've added 'qualifiers' to robolectricConfig and KDoc to clarify.
robolectricConfig will be passed to Robolectric's `@Config` annotation in the generated test class.
If the user makes a mistake, it will result in a compilation error in the generated code.

```
@Config(sdk = [33], qualifiers = RobolectricDeviceQualifiers.Pixel4a)
```

The reason I added this option is that Robolectric's `@Config` has many options, and I thought users might want to set up the package for the Application class or Manifest path, and so on. However, managing all these options would be challenging.

Reply:
Very true. I've also realised the user could also pass the fontScale there (it is the only config that is not passed via the qualifier string). For instance:
`@Config(fontScale = 1.3f)`

I would say that is a Config option we could ignore for now, since I believe not so many people will use it, and add it later, unless you think that is sth trivial to handle. Would create an issue not to forget about it though.

Since there are many options, maybe also makes sense to state in the README.md what options are supported, and even throw an error for those not supported yet.

But up to you, this looks already pretty good
Your call  :)